### PR TITLE
chore: release v2.0.0-beta.4

### PR DESCRIPTION
**Pull request by noelmace**
## release PR

merge = release to production (https://blindnet.dev)
due date: September 22
review is optional - see the list of commits below, and the global state of https://next.blindnet.dev

### CODE REVIEW

File: docs/_partials/_features-list.mdx [hunk 1]
```diff
@@ -0,0 +1,7 @@
+- easy to integrate privacy management
+  - white-label **Web Components** for all privacy settings
+  - helpers libraries for easy **capture** of sensitive data with associated user consent
```
Comment (m4rk055) [inline docs/_partials/_features-list.mdx:3]:
```suggestion
  - helper libraries for easy **capture** of sensitive data with associated user consent
```

File: docs/05-access-management/index.mdx [hunk 2]
```diff
@@ -4,12 +4,48 @@ import WipWarning from '../_partials/_wip.mdx';
...
-  <p>The current version of the access management dashboard is only provided to support our standalone <a href="/docs/encryption/">encryption libraries</a>.</p>
-  <p>A new dashboard allowing you to manage all aspects of blindnet devkit applications and users, including encryption, will be soon available.</p>
-  <p>In the mean time, we provide a <a href="https://devkit-pce-staging.azurewebsites.net/swagger/">pre-configured Privacy Computation Engine</a> to be used directly or via other components of the DevKit without any need for an application id. Check out our <a href="../tutorial">Quick Start Tutorial</a> for more details.</p>
+## Set Up your Privacy System
+
+<WipWarning callToAction="false">
+  <p>The instruction in this section are only here to give you a glips on how the system identification works with the Privacy Computation Engine.</p>
+  <p>At this moment, one blindnet devkit install using a single instance of the Privacy Computation Engine <em>is</em> considered as a coherent single system automatically, using an internally generated ID.</p>
+  <p>Check out out <a href="/docs/tutorial">introductory tutorial</a> for a concrete and up-to-date overview.</p>
```
Comment (m4rk055) [inline docs/05-access-management/index.mdx:12]:
```suggestion
  <p>Check out our <a href="/docs/tutorial">introductory tutorial</a> for a concrete and up-to-date overview.</p>
```

File: docs/05-access-management/index.mdx [hunk 3]
```diff
@@ -4,12 +4,48 @@ import WipWarning from '../_partials/_wip.mdx';
...
+
+[Contact us](https://www.blindnet.io/contact) directly to book a demo and discuss how our new dashboard can fit your specific needs.
+
+:::
+
+The blindnet dashboard is a service and interface engineered to help you accomplish management operations intuitively from a central place.
+
+Futures include (but aren't limited to):
+
+- managing all users and applications via an intuitive interface
```
Comment (m4rk055) [inline docs/05-access-management/index.mdx:43]: User - dashboard users, not application users?
Application users are managed by DCI.

File: docs/05-access-management/index.mdx [hunk 4]
```diff
@@ -4,12 +4,48 @@ import WipWarning from '../_partials/_wip.mdx';
...
 
-<WipWarning>
-  <p>The current version of the access management dashboard is only provided to support our standalone <a href="/docs/encryption/">encryption libraries</a>.</p>
-  <p>A new dashboard allowing you to manage all aspects of blindnet devkit applications and users, including encryption, will be soon available.</p>
-  <p>In the mean time, we provide a <a href="https://devkit-pce-staging.azurewebsites.net/swagger/">pre-configured Privacy Computation Engine</a> to be used directly or via other components of the DevKit without any need for an application id. Check out our <a href="../tutorial">Quick Start Tutorial</a> for more details.</p>
+## Set Up your Privacy System
+
+<WipWarning callToAction="false">
+  <p>The instruction in this section are only here to give you a glips on how the system identification works with the Privacy Computation Engine.</p>
+  <p>At this moment, one blindnet devkit install using a single instance of the Privacy Computation Engine <em>is</em> considered as a coherent single system automatically, using an internally generated ID.</p>
```
Comment (m4rk055) [inline docs/05-access-management/index.mdx:11]: This statement is hard to understand (although it fully explains the point). Split it into multiple sentences.

File: docs/05-access-management/index.mdx [hunk 5]
```diff
@@ -4,12 +4,48 @@ import WipWarning from '../_partials/_wip.mdx';
...
+  <p>Check out out <a href="/docs/tutorial">introductory tutorial</a> for a concrete and up-to-date overview.</p>
 </WipWarning>
 
----
+In blindnet devkit, all components are identified to be part of a coherent "system" using a "System ID".
```
Comment (m4rk055) [inline docs/05-access-management/index.mdx:15]: Mention that components belonging to a single system id easily communicate, while it's hard for components of different systems (only through transfers and privacy requests having `target` specified). So it's the best to have one devkit system id for one logical system (that encapsulates the data).

**Review by m4rk055 — CHANGES_REQUESTED** [review]

File: docs/05-access-management/index.mdx (hunk 3 above)
Comment (noelmace) [reply docs/05-access-management/index.mdx:43]: Users != Data Subjets
Here, I'm thinking administration (access to the different tools and apps), not privacy request management.

**Review by m4rk055 — DISMISSED** [review]

**Pull request merged**